Dependencies/xr/Source/OpenXR/XR.cpp

```diff
@@ -866,4 +870,9 @@ namespace xr
     {
         throw std::runtime_error("Anchors not yet implemented for OpenXR.");
     }
+
+    void System::Session::SetPlaneDetectionEnabled(bool) const
+    {
+        // NYI
```

Comment:
```suggestion
        assert(!enabled);
```

Reply:
`NYI` is, "Not yet implemented," I'm guessing? You can if you want to, but no need to call that out, I think; there's a lot that's not implemented. 😆